ops/ccs-ops-misc/load_test/README.md

```diff
@@ -47,10 +47,10 @@ See: http://docs.locust.io/en/stable/installation.html
 
 
 In addition to these libraries, you'll also need to copy a PEM (credentials) file and set the user to your SSH username on the test box. 
-Run the following two commands on the box:
+Run the following two commands on the box from your local directory:
 
 ```
-sudo cp /usr/local/bfd-server/bluebutton-backend-test-data-server-client-test-keypair.pem ~/
+sudo cp /usr/local/bfd-server/bluebutton-backend-test-data-server-client-test-keypair.pem ./
```

Comment:
I think this one was OK before but if changing this I think the command below on line 70 and the comment on line 76 should change since clientCertPath would now be wherever they ran this command from.

Reply:
Actually since this isn't python yeah I think I can leave it as ~/ for this.